docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -1,15 +1,15 @@
 # GSoC 2024. Gradle Build Server – support for Android projects
 
-This project aims to enhance [Build Server for Gradle](https://github.com/microsoft/build-server-for-gradle) from Microsoft by integrating powerful Android Studio features. Bridging the gap between Android Studio and [Build Server for Gradle](https://github.com/microsoft/build-server-for-gradle), will significantly improve the development experience for many Android developers using IDEs based on [Build Server Protocol (BSP)](https://build-server-protocol.github.io).
+This project aims to enhance [Gradle Build Server](https://github.com/microsoft/build-server-for-gradle) project from Microsoft by integrating powerful Android Studio features. Bridging the build process gap between [Android Studio](https://developer.android.com/studio) and [Gradle Build Server](https://github.com/microsoft/build-server-for-gradle), will significantly improve the development experience for many Android developers using Text Editors/IDEs which utilize the [Build Server Protocol (BSP)](https://build-server-protocol.github.io).
 
 ## Status
-Accepted
+Passed mid-term evaluation
 
 ## Goal
```

Comment:
Some status summary here would be nice. E.g. DONE / In_PROGRESS / PLANNED / CANCELLED . It is fine without it too, you cave it below

Reply:
Fixed in https://github.com/gradle/community/pull/53/commits/2dff07fafdaa6178b1959e3f2e28aff1572fe6df